numaprom/entities.py

```diff
@@ -14,54 +19,45 @@ class Status(str, Enum):
     POST_PROCESSED = "post_processed"
 
 
-@dataclass_json
-@dataclass
+@dataclass(frozen=True)
 class Metric:
     timestamp: str
-    value: float = 0
+    value: float
 
 
-@dataclass_json
 @dataclass
-class Payload:
+class StreamPayload:
     uuid: str
-    metric_name: str
-    key_map: Dict
-    src_labels: Dict[str, str]
-    processedMetrics: List[Metric]
-    startTS: str
-    endTS: str
+    win_arr: Matrix
+    win_ts_arr: List[str]
```

Comment:
what is the difference between win_arr and win_ts_arr?

Reply:
`win_arr` is only for the actual time series. Can take in multivariate too.
`win_ts_arr` is the timestamp array

Reply:
we can't have them in one?